ansible/roles/hypervisor/tasks/main.yml

```diff
@@ -13,6 +13,8 @@
   command: virsh snapshot-revert --domain {{perf_vm_domain}} --snapshotname='readyfortestsnap' --running
   when:
     - perf_vm_domain == "CP" or (perf_vm_domain == "candlepin-perf-mysql" and target_branch == "master")
+      or (perf_vm_domain == "candlepin-perf-mysql"
```

Comment:
imo this is no longer a scalable approach. Please rename 'readyfortestsnap' snapshot to 'master_hotfix', and consider using something like:

`- name: Reset snapshot on performance vm
  command: virsh snapshot-revert --domain {{perf_vm_domain}} --snapshotname='{{target_branch}}-snapshot' --running`
  
`tags:`
`    - skip_ansible_lint`

 I would have rather we would have done something like this when we had two snapshots, but now it seems unavoidable.